contracts/governed_token_wrapper/lib.rs

```diff
@@ -71,6 +71,8 @@ mod governed_token_wrapper {
         InvalidValueSentForWrapping,
         /// Invalid token address
         InvalidTokenAddress,
+        /// Invalid token address
+        ValidTokenAddress,
```

Comment:
Remove the duplicates

Reply:
it's actually different, i have fix the doc now